semeio/jobs/rft/utility.py

```diff
@@ -0,0 +1,265 @@
+import os
+import datetime
+import argparse
+
+from ecl.rft import EclRFTFile
+from ecl.grid import EclGrid
+from semeio.jobs.rft.zonemap import ZoneMap
+
+
+def strip_comments(line):
+    return line.partition("--")[0].rstrip()
+
+
+def existing_directory(path):
+    if not os.path.exists(path):
+        raise argparse.ArgumentTypeError("The path {} does not exist".format(path))
+    return path
+
+
+def load_zonemap(path):
```

Comment:
Why is this not a function of `ZoneMap`

Reply:
When we started this, all the file parser were indeed part of their corresponding classes. So the `load_zonemap` was a class function of `ZoneMap`. My personal opinion is that I don't think the peculiarity of the file formats that exist today should be in any of the classes. So instead I thought it would be better design to have all the parsers in one location, and have the classes be more general.

Agree with my thoughts?

Reply:
After some discussions we figured it is better to keep the parsing logic within each classes. Moved functionality